pkgs/development/libraries/soci/default.nix

```diff
@@ -33,7 +33,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     cmake
   ] ++ lib.optionals stdenv.isDarwin (
-    darwin.apple_sdk_11_0.frameworks.Kerberos
+    builtins.attrValues {
```

Comment:
@wegank with this it passes. Is it preferrable to have it this way or declare this input via `darwin_kerberos = `builtins.attValues...` before `stdenv.mkDerivation` via `let`?

Reply:
just thinking that defining this framework directly in`nativeBuildInput` makes it explicit/easy to follow and does only load when `stenv.isDarwin`. Since I am new to nix and don't now yet about many conventions I wonder if what I do here is elegant, or if there are even better ways.

Reply:
I think it's more common to put `inherit (darwin.apple_sdk_11_0.frameworks) Kerberos;` before `stdenv.mkDerivation` and to call `Kerberos` here. Also, this should go to `buildInputs`, as they're frameworks native to host platform.

Reply:
> I think it's more common in nixpkgs to put `inherit (darwin.apple_sdk_11_0.frameworks) Kerberos;` before `stdenv.mkDerivation` and to call `Kerberos` here. Also, this should go to `buildInputs`, as they're frameworks native to host platform instead of build platform.

thanks a lot for this clarification, I will adapt accordingly.